stripe/src/main/java/com/stripe/android/model/SourceCodeVerification.java

```diff
@@ -23,7 +23,7 @@
  */
 public class SourceCodeVerification extends StripeJsonModel {
 
-    // Note: these are the same as the values for the @Redirect.Status StringDef.
+    // Note: these are the same as the values for the @Redirect.RequiredAddressFields StringDef.
```

Comment:
Yeah, this shouldn't be here. I think Android Studio did a refactor for me on the word "Status"